relays/substrate/src/rialto_millau/mod.rs

```diff
@@ -115,6 +115,9 @@ async fn run_init_bridge(command: cli::InitBridge) -> Result<(), String> {
 				.next_account_index(millau_sign.signer.public().into())
 				.await?;
 
+			// at Westend -> Millau initialization we're not using sudo, because otherwise our deployments
+			// may fail, because we need to initialize both Rialto -> Millau and Westend -> Millau bridge.
+			// => since there's single possible sudo account, one of transaction may fail with duplicate nonce error
```

Comment:
Ugh, that sounds a bit like a nasty hack. Maybe let's set up an owner for all netwoks and never use `sudo` (sounds reasonable to me, cause it's one runtime requirement less). Please add an issue for this if it makes sense for you too.

Reply:
No, I don't see this particular code as a hack. The issue is how we start bridges - i.e. simultaneously, but I don't see a way on how to improve that now.

Reply:
I don't see this as a hack either. We're just deciding to use a different owner than Sudo in this particular case (because of nonce issues)

Reply:
We had to put at least 5 comments to justify why we are not using `sudo` here (and when we set the owner).
So what I'm proposing is that instead of special casing, maybe let's never use sudo and just assign owners to specific bridges? Then we don't have to justify that. Is there a compelling reason to keep `sudo` for other bridge initialisations?

Reply:
Well the main problem here isn't the use of `Sudo` per say, but the use of `Alice`, who happens to be `Sudo` for all our local/development networks.

I agree though, it could make sense to have different non-Sudo accounts set as owners instead.